src/vp8_probs.h

```diff
@@ -336,4 +336,240 @@ static const int vp8_bits_per_mb[2][128] =
     }
 };
 
+static const unsigned short vp8_prob_cost[256] =
```

Comment:
Yes. There is no problem when it is included by multi-files.
But if the header file is included by file A and B,  A and B have one copy of the above static tables.
They are duplicated.  But if they are declared by using external and be split into *.h/*.c, the duplication can be avoided.